core/src/main/java/com/novoda/noplayer/drm/provision/ModularDrmKeyRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.novoda.noplayer.drm.provision;
+
+public class ModularDrmKeyRequest {
+
+    private final String url;
+    private final byte[] data;
+
+    public ModularDrmKeyRequest(String url, byte[] data) {
+        this.url = url;
+        this.data = data;
+    }
+
+    public String getUrl() {
```

Comment:
Also, add `toString` and `hashcode` I don't like reading gibberish when I get a test crash 😄